addons/mail/static/src/core/common/chat_window.scss

```diff
@@ -1,9 +1,9 @@
 .o-mail-ChatWindow {
     width: $o-mail-ChatWindow-width;
 
-    z-index: 999; // messaging menu is dropdown (1000)
+    z-index: 1001; // messaging menu is 1000
     &.o-mobile {
-        z-index: 1001; // above messaging menu (chat window takes whole screen)
+        z-index: 1002; // above messaging menu (chat window takes whole screen)
```

Comment:
Could use `$zindex-dropdown + 1` for `o-mail-ChatWindow`, as the problematic SCSS rule in documents use that.
So any change of value to `$zindex-dropdown` could ensure this works the same.

The intent for this value is to be below messaging menu zindex, which is ok because this is 1070 from popover.
The chat windows in mobile, based on comment, seems to want them be above messaging menu, but this wasn't the case as `1070 < 1001` previously. This "worked" because clicking on an item in the messaging menu actually closes the messaging menu. The messaging menu as thus actually precedence over chat window.

This means this mobile rule is unnecessary, so I think we can just drop it.